bin/_mocha

```diff
@@ -68,8 +68,8 @@ program
   .option('-S, --sort', "sort test files")
   .option('-b, --bail', "bail after first test failure")
   .option('-d, --debug', "enable node's debugger, synonym for node --debug")
-  .option('-g, --grep <pattern>', 'only run tests matching <pattern>')
-  .option('-f, --fgrep <string>', 'only run tests containing <string>')
+  .option('-g, --grep <pattern>', 'only run tests having full title matching <pattern>')
+  .option('-f, --fgrep <string>', 'only run tests having full title containing <string>')
```

Comment:
> If you want exact matching you have to use `^full title$'.

Yep! That’s exactly what I’m trying to make a bit clearer with this wording change: the pattern given to `--grep` regards the full test title, and this is especially valuable to understand in cases like the one you brought up in #2360.

> Also this PR doesn't change the way the matching works.

Exactly! It only slightly changes how the full test title is computed, which means it changes the string that in the end is matched against the given pattern; the matching doesn’t need to change as far as I understand. 🤓